### ISSUE TITLE
Webpack fails to build bundle because of missing Jquery dependency

### ISSUE THREAD

**Issue by vinaynb** [STUkh/generator-angular-webpack-es6#19]
I setup the generator globally and then created the project using `yo angular-webpack-es6`. I chose to install oclazyload module but skipped Jquery. When starting the dev server via `npm run dev` webpack bundling was stuck at 26% and there were errors of missing module jquery which was supposedly a dependency of oclazyload (because the paths in error message indicate that).

If oclazyloading does indeed require Jquery then it should be installed by default as due to this the project fails to kickoff.

Screenshot of error
![screenshot from 2017-04-27 16 00 47](https://cloud.githubusercontent.com/assets/1573465/25479525/bc0d92a0-2b62-11e7-96a6-8d6ab5e327b6.png)

I installed jquery using npm install jquery --save and the error went away and bundle was built successfully.

**Comment by vinaynb** [comment]
@STUkh can you have a look at this ?

**Comment by vinaynb** [comment]
**UPDATE**
It requires jquery as dependency even though you do not include ocLazyLoad plugin.
Related issue #23

**Comment by vinaynb** [comment]
I found the issue. It is due to the below code in config/webpack/global.js
```
new webpack.ProvidePlugin({
           $: 'jquery',
           jQuery: 'jquery',
           'window.jQuery': 'jquery',
           'window.jquery': 'jquery'
       })
```
ProvidePlugin is used to access jQuery directly in the source code without the need of import statements. Due to this plugin it expects jQuery node_module to be installed which is not by default. Hence i guess this needs to be removed or be made optional if user chooses to install jQuery during yeoman prompts.

For now you can comment the above code and run `npm start`

**Comment by niklaszantner** [comment]
FYI, the path to the global webpack config has changed to: `config/webpack/global.js`.

**Comment by vinaynb** [comment]
Thanks for the tip. Updated !

**Comment by STUkh** [comment]
Fixed in 1.0.6
